cmake/modules/extensions.cmake

```diff
@@ -5412,21 +5412,37 @@ function(add_llext_target target_name)
   # expected `pkg_input` location. If actual POST_BUILD commands are defined,
   # they will take care of this and the default copy is replaced by a no-op.
   set(has_post_build_cmds "$<TARGET_PROPERTY:${llext_proc_target},has_post_build_cmds>")
+
   set(noop_cmd ${CMAKE_COMMAND} -E true)
   set(copy_cmd ${CMAKE_COMMAND} -E copy ${llext_lib_output} ${llext_pkg_input})
+
```

Comment:
Whitespace leftover from previous iteration?

Reply:
Indeed. Removing it.